keyless/pepper/common/src/vuf/bls12381_g1_bls.rs

```diff
@@ -14,19 +18,131 @@ use ark_std::{
     rand::{CryptoRng, RngCore},
     UniformRand,
 };
+use blstrs::{self, Compress};
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+use serde_big_array::BigArray;
 use std::ops::Mul;
 
 pub struct Bls12381G1Bls {}
 
-pub static DST: &[u8] = b"APTOS_PEPPER_SERVICE_BLS12381_VUF_DST";
+pub static DST: &[u8] = b"APTOS_PEPPER_BLS12381_VUF_DST";
+
+pub static PINKAS_DST: &[u8] = b"APTOS_PEPPER_PINKAS_VUF_DST";
+
+pub static PINKAS_SECRET_KEY_BASE: &[u8] = b"APTOS_PEPPER_PINKAS_VUF_SECRET_KEY_BASE";
+
+pub static PINKAS_SECRET_KEY_BASE_AFFINE: Lazy<G2Affine> =
+    Lazy::new(|| Bls12381G1Bls::hash_to_g2(PINKAS_SECRET_KEY_BASE, PINKAS_DST));
+
+pub static PINKAS_SECRET_KEY_BASE_G2: Lazy<blstrs::G2Projective> =
```

Comment:
Why do you need `PINKAS_SECRET_KEY_BASE_AFFINE` and `PINKAS_SECRET_KEY_BASE_G2`? They are the same point in two different library formats...

These things are confusing & should be documented, if we are gonna keep them...

Reply:
I don't, i just hadn't committed to a library yet.  Was testing both out.